frontend/public/components/node.tsx

```diff
@@ -54,7 +54,15 @@ const HeaderSearch = props => <ListHeader>
   <ColHead {...props} className="col-md-2 col-sm-3 hidden-xs" sortField="status.addresses">Node Addresses</ColHead>
 </ListHeader>;
 
-const NodeStatus = ({node}) => isNodeReady(node) ? <span className="node-ready"><i className="fa fa-check"></i> Ready</span> : <span className="node-not-ready"><i className="fa fa-minus-circle"></i> Not Ready</span>;
+const NodeStatus = ({node}) => {
+  if (isNodeReady(node)) {
+    return <span className="node-ready"><i className="fa fa-check"></i> Ready</span>;
+  }
+  if (!isNodeSchedulable(node)) {
```

Comment:
It looks like it will short-circuit before it gets here if the node is ready, but not schedulable.

Reply:
Prob the latter seems sufficient?